### PR TITLE
Changed the styling

### PR DESCRIPTION
**Pull request by narufish**
## What
1. Added some styling for the buttons, text input fields and drafts items view.

## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

File: typescript/simple-mercari-web/src/App.css [hunk 1]
```diff
@@ -1,60 +1,207 @@
...
+}
+
+#button:hover {
+  background-color: rgb(119, 26, 23);
+  
+}
+
+#button:active {
+  background-color: #ec6363;
+}
```
Comment (narufish) [inline typescript/simple-mercari-web/src/App.css:221]: added styling for buttons

File: typescript/simple-mercari-web/src/App.css [hunk 2]
```diff
@@ -1,60 +1,207 @@
...
+
+input[type=text] {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+}
```
Comment (narufish) [inline typescript/simple-mercari-web/src/App.css:228]: added styling for input text fields.

File: typescript/simple-mercari-web/src/App.tsx [hunk 3]
```diff
@@ -1,26 +1,71 @@
...
+          reload={reload}
+          onLoadCompleted={ () => setReload(false) }
+          selectModeOn={selectMode.toggled}
+          checkList={isCheck}
+          handleClick={ (id: number, checked: boolean) => newChange(id, checked) } />
       </div>
     </div>
   )
-}
+};
```
Comment (narufish) [inline typescript/simple-mercari-web/src/App.tsx]: ignore this file, this was accidentally added to the commit.

File: typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx [hunk 4]
```diff
@@ -44,21 +50,60 @@ export const ItemList: React.FC<Prop> = (props) => {
...
+                <div className={completeCheck(item.delivery)}></div>
+                <div className={completeCheck(item.price)}></div>
+              </div>
+            </div>
           </div>
         )
       })}
     </div>
   )
-};
```
Comment (narufish) [inline typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx:108]: again, accidentally added to commit. please ignore.

File: typescript/simple-mercari-web/src/components/Listing/Listing.tsx [hunk 5]
```diff
@@ -54,19 +58,26 @@ export const Listing: React.FC<Prop> = (props) => {
...
+	<div> 
+          <label id="button" htmlFor="image">
+            <input type='file' name='image' id='image' onChange={onFileChange} required hidden />
+            Choose file
+          </label>
+          <span></span>
 	</div>
 	<div>
-          <button type='submit'>List this item</button>
+          <button type='submit' id="button">List this item</button>
```
Comment (narufish) [inline typescript/simple-mercari-web/src/components/Listing/Listing.tsx:78]: Mainly, made changes to the buttons for styling.

**Review by kappanjoe — APPROVED** [review]
LGTM!~

**Pull request merged**